ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_snapshot_collection.py

```diff
@@ -212,18 +214,18 @@ def update_snapcoll(
         return (False, False, f"Snapshot collection update failed | {ex}", {})
 
 
-def delete_snapcoll(client_obj, snapcoll_name):
+def delete_snapcoll(client_obj, snapcoll_name, volcoll_name):
 
     if utils.is_null_or_empty(snapcoll_name):
         return (False, False, "Snapshot collection deletion failed as snapshot collection name is not present.", {})
 
     try:
-        snapcoll_resp = client_obj.snapshot_collections.get(id=None, name=snapcoll_name)
+        snapcoll_resp = client_obj.snapshot_collections.get(id=None, name=snapcoll_name, volcoll_name=volcoll_name)
         if utils.is_null_or_empty(snapcoll_resp):
-            return (False, False, f"Snapshot collection '{snapcoll_name}' not present to delete.", {})
+            return (False, False, f"Snapshot collection '{snapcoll_name}' inside volume collection '{volcoll_name}' not present to delete.", {})
```

Comment:
nit: Snapshot collection '{snapcoll_name}' for volume collection '{volcoll_name}' not present to delete.